wire-schema/src/jvmMain/java/com/squareup/wire/schema/EnumType.kt

```diff
@@ -49,12 +51,21 @@ class EnumType private constructor(
     allowAlias = options.get(ALLOW_ALIAS)
   }
 
-  override fun validate(linker: Linker) {
+  override fun validate(linker: Linker, syntax: Syntax?) {
     val linker = linker.withContext(this)
 
     if ("true" != allowAlias) {
       validateTagUniqueness(linker)
     }
+    if (syntax == PROTO_3) {
+      validateTagZero(linker)
+    }
+  }
+
+  private fun validateTagZero(linker: Linker) {
+    if (constants.none { it.tag == 0 }) {
+      linker.addError("missing a tag 0 constant")
```

Comment:
Is `tag` a wire-specific word? Should we use `value zero` like the official doc says?

Reply:
For the error, message we could do something like `missing a zero value at the first element`